lib/Doctrine/MongoDB/Aggregation/Stage/Project.php

```diff
@@ -61,11 +61,15 @@ public function avg($expression1/* , $expression2, ... */)
 
     /**
      * Shorthand method to exclude the _id field.
+     *
+     * @deprecated Deprecated in 1.5, please use @see excludeFields.
```

Comment:
I think you should use `@link` instead - http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.see.pkg.html